lndclient/router_client.go

```diff
@@ -0,0 +1,161 @@
+package lndclient
+
+import (
+	"context"
+	"time"
+
+	"github.com/lightningnetwork/lnd/channeldb"
+	"google.golang.org/grpc/codes"
+
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/status"
+
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/lntypes"
+)
+
+// RouterClient exposes payment functionality.
+type RouterClient interface {
+	SendPayment(ctx context.Context, invoice string,
```

Comment:
Missing `godoc` comments here and below.

Reply:
Added